src/js/msp/MSPHelper.js

```diff
@@ -20,7 +20,7 @@ import { OSD } from "../tabs/osd";
 // Used for LED_STRIP
 const ledDirectionLetters    = ['n', 'e', 's', 'w', 'u', 'd'];      // in LSB bit order
 const ledBaseFunctionLetters = ['c', 'f', 'a', 'l', 's', 'g', 'r']; // in LSB bit
-let ledOverlayLetters        = ['t', 'o', 'b', 'v', 'i', 'w']; // in LSB bit
+let ledOverlayLetters        = ['t', 'y', 'o', 'b', 'v', 'i', 'w']; // in LSB bit
```

Comment:
Needed for backward compatibility

Reply:
I handle it here:
![image](https://user-images.githubusercontent.com/62965528/221978273-8548b778-528b-459c-a3c8-3479aaf06aac.png)